static/sass/ds/6-components/_message.scss

```diff
@@ -0,0 +1,33 @@
+.c-message {
+  position: relative;
+
+  @include mq($until: bp-l) {
+    border-bottom: 2px solid $color-gray-light;
+  }
+
+  &__top {
+    align-items: center;
+    display: flex;
+
+    .c-icon {
+      margin-right: $size-b;
+    }
+  }
+
+  &__close {
+    border: 1px solid $color-white-off;
```

Comment:
Probably deliberate, but just in case I'll note it here: I'm not seeing the border in the mockup, but maybe it came in a later revision

Reply:
Yeah, I thought it looked weird without the border.